Automattic/sensei
# Ensure all strings in block pattern templates are translated and escaped

Fixes #6321 

### Changes proposed in this Pull Request

Set up translations in the Course List block pattern template.

### Testing instructions

1. Set your site's language to Spanish (note that you may need to save Permalinks to flush the cache and get the frontend paths working).
2. Download the latest translations on the Dashboard > Updates page (Escritorio > Actualizaciones)
3. Create three courses. Add two lessons to each.
4. On one course, complete one lesson.
5. On another course, complete all lessons.
6. Do not start the last course.
7. Install Loco Translate. Open the translation file for Sensei LMS in the language you've chosen.
8. Add translations for "Start Course", "Continue", and "Visit Results" and save. This will ensure the `.po`/`.mo` and `.json` files are generated.
9. Create a new page and add the Course List block (Lista del curso).
10. Ensure that in the editor, the buttons are translated (Empezar el curso)
11. Publish the page and view the frontend. You should see all three button statuses, and they should all be translated: Empezar el curso, Continuar, Visitar resultados.
